processor/batchprocessor/batch_processor.go

```diff
@@ -244,17 +245,24 @@ func (bt *batchTraces) add(item interface{}) {
 	td.ResourceSpans().MoveAndAppendTo(bt.traceData.ResourceSpans())
 }
 
-func (bt *batchTraces) export(ctx context.Context, sendBatchMaxSize int) error {
+func (bt *batchTraces) export(ctx context.Context, sendBatchMaxSize int, returnBytes bool) (int, int, error) {
```

Comment:
question: should we do the `stats.Record` call within export rather than returning it? That way we don't need to make any function signature changes?

Reply:
i can see why that would be annoying because then instead of a single stats record call, you have to make one in each batch processor. What do you think?

Reply:
Yeah, I'd rather change the signature, especially since it's only used in one place, than duplicate the stats recording code.